src/troute-network/troute/DataAssimilation.py

```diff
@@ -1198,7 +1209,8 @@ def _create_reservoir_df(data_assimilation_parameters, reservoir_da_parameters,
     crosswalk_lakeID_field = streamflow_da_parameters.get('crosswalk_' + res_source + '_lakeID_field',res_source + '_lake_id')
     qc_threshold           = data_assimilation_parameters.get("qc_threshold",1)
     interpolation_limit    = data_assimilation_parameters.get("interpolation_limit_min",59)
-	
+    LOG.info("_create_canada_df function is started.")
```

Comment:
should this be "create_reservoir_df function..."?

Reply:
For this function in particular, because it can get called twice (once for usgs reservoirs, once for usace reservoirs), it might be better to put this LOG outside of the function. Instead it could go where it gets called for each reservoir type.

Reply:
Or, maybe in the message here it could print which type of reservoir it is creating? That might be a better solution.

Reply:
Its get called either from great_lake class or NudgingDA class. Do we want to specify where it called from?

Reply:
`_create_reservoir_df()`? I should only be called in `PersistenceDA`...

Reply:
ok got it. I thought we are talking about `_create_canada_df()`